cmake/modules/FindJpegTurbo.cmake

```diff
@@ -1,50 +0,0 @@
-# Source: caffe2/cmake/Modules/FindJpegTurbo.cmake
-#
-# - Try to find libjpeg-turbo (more specifically libturbojpeg)
-#
-# The following variables are optionally searched for defaults
-#  JPEG_TURBO_ROOT_DIR:         Base directory where all JPEG_TURBO components are found
-#
-# The following are set after configuration is done:
-#  JpegTurbo_FOUND
-#  JpegTurbo_INCLUDE_DIR
-#  JpegTurbo_LIBRARY
-#  JpegTurbo_VERSION
-
-set(JPEG_TURBO_ROOT_DIR "" CACHE PATH "Folder contains JpegTurbo")
```

Comment:
We need to update docs removing info about JPEG_TURBO_ROOT_DIR

Reply:
I think this file should stay, but it should look for JPEG

Reply:
about that, there is a native package to find jpeg:
https://cmake.org/cmake/help/v3.0/module/FindJPEG.html
Do you think we need a custom one?

Reply:
It definitely doesn't handle the situation when lib is custom build in some custom path. Or does it?

Reply:
It does:
cmake -DJPEG_INCLUDE_DIR=/home/janton/libjpeg-turbo-1.5.3/install/include/ -DJPEG_LIBRARY=/home/janton/libjpeg-turbo-1.5.3/install/lib/libjpeg.so ..